tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def myfunc(a, b, x):
+    """
+
+    :param a:
+        slope
+    :param b:
+        offset
+    :param x:
+    :return:
+    """
+
```

Comment:
Note that this violates [PEP 257](https://github.com/python/peps/blob/master/pep-0257.txt).  "There's no blank line either before or after the docstring."

But as this is not about the contents of the docstring, but is about how the docstring is placed wrt other nodes in the ast, IMO this is definitely beyond scope of the first merge.